templates/_helpers.yaml

```diff
@@ -276,3 +276,89 @@ Create the name of the cleanup service account to use
     {{ default "default" .Values.airflow.cleanup.serviceAccount.name }}
 {{- end -}}
 {{- end -}}
+
+
+{{ define "cleanup_task_metadata_airflow_config_path" -}}
```

Comment:
Why not use `airflow_config_path`?

Reply:
its fixed now - removed the additional helpers added . Instead called the subchart directly

Reply:
```{{ template "airflow_config_path" .Subcharts.airflow }}```  this resolved it